documentation/docs.css

```diff
@@ -0,0 +1,190 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
+body{
+    font-family: 'Montserrat', sans-serif;
+    scroll-behavior: smooth;
+}
+:root{
+    --component-library-name-font : 2.5rem;
+    --cl-color : #4d4dff;
+    --sidebar-background-color:white;
+    --sidebar-text-color:white;
+}
+*{
+    margin: 0;
+    padding: 0;
```

Comment:
i think box-sizing : border-box. is good to have.

Reply:
Yeah . I'll do that